azure-pipelines.yml

```diff
@@ -0,0 +1,96 @@
+variables:
+  dockerVersion: '19.03.5'
+  dotnetVersion: '3.1.x'
+
+trigger:
+- master
+- development
+
+pool:
+  vmImage: windows-2019
+
+jobs:
+- job: build
+  displayName: Build app and run tests
+
+  steps:
+  - task: MSBuild@1
+    displayName: 'Build GPITFutures ISAPI Database'
+    inputs:
+      solution: '**/*.sqlproj'
+      clean: true
+
+  - task: CopyFiles@2
+    inputs:
+      SourceFolder: '$(agent.builddirectory)'
+      Contents: '**'
+      TargetFolder: '$(build.artifactstagingdirectory)\sql'
+      CleanTargetFolder: true
+
+  - task: PublishBuildArtifacts@1
+    inputs:
+      PathtoPublish: '$(Build.ArtifactStagingDirectory)\sql'
+      ArtifactName: 'sql'
+      publishLocation: 'Container'
+
+- job: dockerBuildAndPush
+  displayName: Build and push docker containers to the acr
+
+  pool:
+    vmImage: 'ubuntu-latest'
+
+  steps:
+
+  - task: UseDotNet@2
+    displayName: 'Use DotNet Core 3.1'
+    inputs:
+      version: 3.1.x
```

Comment:
I think you must've forgotten to replace the version here with the variable you declared on top.
instead of `version: 3.1.x`
I think it should be: `version: $(dotnetVersion)`